projects/m1/032-taxi-fare/js/index.js

```diff
@@ -0,0 +1,12 @@
+function taxiFare(distance) {
+    const basicFare = 4;
+    let myDistance = (distance / 140) * 0.25;
+    let globalFare = myDistance + basicFare;
```

Comment:
Be careful with `let` vs `const`.
A variable must be declared `let` if it **changes after its declaration**, not if it is evaluated during the assignment of a result, even if it comes from a mathematical expression involving other variables.